apps/financial-aid/backend/migrations/20211110191445-aid-municipality-make-municipality_id-unique.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.changeColumn('municipality', 'municipality_id', {
+          type: Sequelize.STRING,
```

Comment:
Could we change this type while we are at it? 🤗

Reply:
what type should this be? Number?  🤠